ruby/tests/encode_decode_test.rb

```diff
@@ -84,6 +84,34 @@ def test_encode_json
     )
 
     assert_match 'optional_int32', json
+
+
+    # Test for enums printing as ints.
```

Comment:
thank @spiffyy99 for the tests https://github.com/protocolbuffers/protobuf/pull/6384